lib/uuid/Kconfig

```diff
@@ -0,0 +1,36 @@
+# Copyright (c) 2024, SECO Mind Srl
+#
+# SPDX-License-Identifier: Apache-2.0
+
+menu "Universally Unique Identifier (UUID)"
+
+config UUID
+	bool "UUID support"
+	help
+	  Enable use of the UUID library.
+
+config UUID_V4
+	bool "UUID version 4 generation support"
+	depends on UUID
+	depends on ENTROPY_GENERATOR
```

Comment:
Is `ENTROPY_GENERATOR` required for this?

As far as I cant tell `uuid_generate_v4` only relies on `sys_rand_get` and I don't see any dependencies on that function.

Reply:
I think it is, without setting `ENTROPY_GENERATOR` I get a linking error when I build for most boards I tried. The error is an undefined reference to `z_impl_sys_rand_get`.

Reply:
Okay, yeah that sort of makes sense. It's just an undocumented requirement of `sys_rand_get` it looks like :)